src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -1860,7 +1860,7 @@ private function addRateLimiterSection(ArrayNodeDefinition $rootNode)
                             ->arrayPrototype()
                                 ->children()
                                     ->scalarNode('lock_factory')
-                                        ->info('The service ID of the lock factory used by this limiter')
+                                        ->info('The service ID of the lock factory used by this limiter (or null to disable locking)')
```

Comment:
If a user used `null` before, they would get an error, which is fine. 

This is not a BC break.